kytos/core/api_server.py

```diff
@@ -59,6 +59,24 @@ def register_rest_endpoint(self, url, function, methods):
             url = url[1:]
         self._start_endpoint(f'/kytos/{url}', function, methods=methods)
 
+    def remove_rest_endpoints(self, napp_id):
+        """Remove a rest endpoint from flask.
+
+        This method will remove all endpoints that contain the napp_id.
+
+        Args:
+            napp_id: String using the format '<username>/<napp_name>'
+
+        """
+        for index, rule in enumerate(self.app.url_map.iter_rules()):
+            if napp_id in rule.rule:
+                del self.app.view_functions[rule.endpoint]
+                # pylint: disable=protected-access
+                self.app.url_map._rules.pop(index)
+                # pylint: enable=protected-access
+
+        self.log.info(f'The Rest endpoints form {napp_id} was disabled.')
```

Comment:
`s/form/from/`
`s/was/were/`

Reply:
Please, use the same message style as when starting the endpoint.